File: compile-mix-test/src/test/java/example/GenerationTest.java
package example;

import static org.junit.jupiter.api.Assertions.assertNotNull;

import org.junit.jupiter.api.Test;

public class GenerationTest {

  @Test
  public void test() {
    assertNotNull(_Address.class);
    assertNotNull(_Employee.class);
    assertNotNull(AddressDaoImpl.class);
    assertNotNull(EmployeeDaoImpl.class);
  }
}

File: compile-kotlin-test/src/test/kotlin/example/GenerationTest.kt
package example

import org.junit.jupiter.api.Assertions.assertNotNull
import org.junit.jupiter.api.Test

class GenerationTest {
    @Test
    fun test() {
        assertNotNull(_Employee::class.java)
        assertNotNull(EmployeeDaoImpl::class.java)
    }
}
